libs/community/langchain_community/vectorstores/chroma.py

```diff
@@ -276,7 +276,7 @@ def add_texts(
         embeddings = None
         texts = list(texts)
         if self._embedding_function is not None:
-            embeddings = self._embedding_function.embed_documents(texts)
+            embeddings = self._embedding_function(texts)
```

Comment:
I don't think this will generally work, as `Embeddings` does not generally implement __call__.